src/main/kotlin/blackJack/domain/Cards.kt

```diff
@@ -0,0 +1,43 @@
+package blackJack.domain
+
+import blackJack.error.ErrorMessage
+
+class Cards(cards: MutableList<Card>) {
+
+    private val _cards: MutableList<Card> = cards.toMutableList()
+
+    val cards: List<Card>
+        get() = _cards.toList()
+
```

Comment:
```suggestion
class Cards(cards: List<Card>) {

    private val _cards: MutableList<Card> = cards.toMutableList()

    val cards: List<Card>
        get() = _cards.toList()
```

backing_property 사용 잘 해주셨어요! 단, 불변으로 받아서 내부에서 가변으로 만들어주면 되지 않을까하네요~